src/components/Movie/SimilarMovies.js

```diff
@@ -0,0 +1,23 @@
+import React, {Component} from 'react';
+
+import MovieCard from './MovieCard';
+
+const SimilarMovies = (props) => {
+    return (
+        <div className="container mt-2">
+            <div className="card">
+                <div className="card-body">
+                    Films by {props.genre} genre
+                </div>
+            </div>
+            <div className="row">
+                { 
+                    props.movies.map(movie => {

```

Comment:
When you're using an arrow function, and you just want to return a value, for brevity you can use this way:

```javascript
 props.movie.map(movie => (<MovieCard key={movie.id} movie={movie} />);
```